sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ObjectShouldBeInitializedCorrectlyBase.cs

```diff
@@ -68,6 +76,8 @@ protected override void Initialize(SonarAnalysisContext context)
                         return;
                     }
 
+                    CompilationAction(ccc.Compilation);
```

Comment:
I usually like the method names to contain verbs since I think they better describe an action. Can we rename **CompilationAction** with **HandleCompilationStart**?

Reply:
good point, I didn't come back to this name and improve it.

it wouldn't be "Compilation Start" because that would induce confusion with `RegisterCompilationStartAction`, this is just a hook to get the Compilation context (it's not related to the compilation phase per se)

I'll try to find a better approach